testing/src/cypress/page-objects/Institution-objects/ManageLocationObject.ts

```diff
@@ -122,4 +198,46 @@ export default class ManageLocationObject {
   phoneNumberErrorMessage() {
     return cy.contains("Phone Number is required");
   }
+
+  locationNameText() {
+    return cy.contains("Location name");
+  }
+  institutionCodeText() {
```

Comment:
Try to keep the consistency between the methods declarations. Please keep a single blank line between then.

Reply:
updated